source/clear-linux/get-started/virtual-machine-install/vmware-esxi-preconfigured-cl-image.rst

```diff
@@ -39,6 +38,8 @@ this command:
 
    curl -O https://download.clearlinux.org/image/clear-$(curl https://download.clearlinux.org/latest)-vmware.vmdk.xz
 
+Visit :ref:`image-types` for additional information about all available |CL| images.
+
 .. include:: ../../guides/maintenance/download-verify-uncompress-linux.rst
    :Start-after: verify-linux:
    :end-before: To uncompress a GZ
```

Comment:
At bottom of page, can you please change "See also" to our new "Related Topics"?